source/documentation/administration_guide/topics/Cluster_Migration_Policy_Settings_Explained.adoc

```diff
@@ -50,3 +50,21 @@ The resilience policy defines how the virtual machines are prioritized in the mi
 |*Migrate only Highly Available Virtual Machines* |Migrates only highly available virtual machines to prevent overloading other hosts.
 |*Do Not Migrate Virtual Machines* |Prevents virtual machines from being migrated.
 |===
+
+.Additional Properties Settings
+[options="header"]
+|===
+|Field |Description/Action
+|*Enable Migration Encryption* a|Allows the virtual machine to be encrypted during migration.
+
+* Cluster default (Don’t encrypt)
+* Encrypt
+* Don’t encrypt
+|*Parallel Migrations* a|Allows you to specify whether and how many parallel migration connections to use.
+
+* *Disabled*: Parallel migrations are disabled.
```

Comment:
this seems a bit ambiguous - we do allow to migrate several VMs in parallel when "Parallel Migration" is disabled
how about something like "The virtual machine would be migrated using a single, non-parallel, connection" ?

Reply:
Change complete.